doc/network-zones.md

```diff
@@ -11,32 +11,21 @@ Each network can be related to up to 3 zones for:
  - IPv4 reverse DNS records
  - IPv6 reverse DNS records
 
-This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4`
-and `dns.zone.reverse.ipv6` in network configuration. LXD will then be
-automatically managing forward and reverse records for all instances,
-network gateways and downstream network ports.
+This is controlled through `dns.zone.forward`, `dns.zone.reverse.ipv4` and `dns.zone.reverse.ipv6` in network configuration.
+LXD will then be automatically managing forward and reverse records for all instances, network gateways and downstream network ports.
 
-To enable the built-in DNS server, `core.dns_address` must be set in the
-server configuration.
+To enable the built-in DNS server, `core.dns_address` must be set in the server configuration.
 
-The built-in DNS server only supports zone transfers through AXFR, it
-cannot be directly queried for DNS records. This means that this feature
-expects the use of an external DNS server (bind9, nsd, ...) which will
-transfer the entire zone from LXD, refresh it upon expiry and provide
-authoritative answers to DNS requests.
+The built-in DNS server only supports zone transfers through AXFR, it cannot be directly queried for DNS records.
+This means that this feature expects the use of an external DNS server (bind9, nsd, ...) which will transfer the entire zone from LXD, refresh it upon expiry and provide authoritative answers to DNS requests.
```

Comment:
@ru-fu are you making these lines long because it affects the display in the end-user facing docs? 
If not, keeping them shorter is more useful when reviewing the PR as it keeps within the github viewport saving having to horizontally scroll back and forth.

Reply:
@tomp The idea is to have one sentence per line to make it easier to see the differences. When updating the docs (thus not rewriting as in this case here ;) ), we'll usually do changes on a sentence basis. If you have a paragraph with random line breaks, GitHub will display loads of "changes" that are really just words moved to a different line. With one line per sentence, that line either has changes or it doesn't.
On the Desktop, GitHub automatically breaks lines as required in the diff - is that different in mobile view?

Reply:
OK makes sense, no worries. The recently added new wider github viewport helps to avoid this happening too often for it to be a problem.